docs/csharp/programming-guide/concepts/linq/toc.yml

```diff
@@ -1,8 +1,6 @@
 - name: Language-Integrated Query (LINQ)
```

Comment:
This is awkward. Because this node has an associated URL, the items are peers rather than children. Better would be to make the first item "Language Integrated Query (LINQ) overview" and have that link to index.md.